client/blindbox/requests/secure_session.py

```diff
@@ -184,19 +185,24 @@ def attestation(self, policy):
 
         # Add further checks for issued at time (iat) and issuer (iss) 
         if payload["x-ms-attestation-type"] != "sevsnpvm":
-            print("Attestation validation failed (not sev-snp report). Exiting.")
+            raise AttestationException("IncorrectAttestation","Attestation validation failed (not sev-snp report). Exiting.")
+            #print("Attestation validation failed (not sev-snp report). Exiting.")
             exit()
         if payload["x-ms-compliance-status"] != "azure-compliant-uvm":
-            print("Attestation validation failed (non-compliant uvm). Exiting.")
+            raise AttestationException("NonCompliantUvm","Attestation validation failed (non-compliant uvm). Exiting.")
+            #print("Attestation validation failed (non-compliant uvm). Exiting.")
             exit()
         if payload["x-ms-sevsnpvm-hostdata"] != cce_policy:
-            print("Attestation validation failed (cce policy mismatch). Exiting.")
+            raise AttestationException("IncorrectPolicy","Attestation validation failed (cce policy mismatch). Exiting.")
+            #print("Attestation validation failed (cce policy mismatch). Exiting.")
             exit()
         if payload["x-ms-runtime"] != nonce:
-            print("Attestation validation failed (nonce mismatch). Exiting.")
+            raise AttestationException("IncorrectNonce","Attestation validation failed (nonce mismatch). Exiting.")
+            #print("Attestation validation failed (nonce mismatch). Exiting.")
             exit()
         if payload["x-ms-sevsnpvm-is-debuggable"] == "false":
-            print("Attestation validation failed (VM is in debug mode). Exiting.")
+            raise AttestationException("DebugMode","Attestation validation failed (VM is in debug mode). Exiting.")
+            #print("Attestation validation failed (VM is in debug mode). Exiting.")
```

Comment:
Oh also, remove those comments when you have the change

Reply:
Done